includes/menu.inc

```diff
@@ -18,36 +18,33 @@ function oe_bootstrap_theme_preprocess_menu(&$variables, $hook) {
   $items = $variables['items'];
   foreach ($items as $key => $item) {
     $class = $item['in_active_trail'] ? ['active'] : [];
-    $menu_item = [
-      'label' => $item['title'],
-      'path' => $item['url']->toString(),
-      'attributes' => new Attribute(['class' => $class]),
-    ];
-    $variables['items'][$key] = $menu_item;
+    $variables['items'][$key]['label'] = $item['title'];
+    $variables['items'][$key]['path'] = $item['url']->toString();
+    $variables['items'][$key]['attributes'] = new Attribute(['class' => $class]);
 
     if (empty($item['below'])) {
       continue;
     }
 
     $class[] = 'nav-link';
-    $menu_item['attributes'] = new Attribute(['class' => $class]);
     $item_with_submenu = [
       'standalone' => TRUE,
       'dropdown' => TRUE,
       'link' => TRUE,
       'alignment' => 'dropdown-menu-sm-start',
-      'trigger' => $menu_item,
+      'trigger' => [
+        'label' => $item['title'],
+        'path' => $item['url']->toString(),
+        'attributes' => new Attribute(['class' => $class]),
+      ],
     ];
-    $variables['items'][$key] = $item_with_submenu;
+    $variables['items'][$key] += $item_with_submenu;
 
     foreach ($item['below'] as $key_sub_item => $sub_item) {
       $subclass = $sub_item['in_active_trail'] ? ['active'] : [];
-      $sub_item_menu = [
-        'label' => $sub_item['title'],
-        'path' => $sub_item['url']->toString(),
-        'attributes' => new Attribute(['class' => $subclass]),
-      ];
-      $variables['items'][$key]['items'][$key_sub_item] = $sub_item_menu;
+      $variables['items'][$key]['items'][$key_sub_item]['label'] = $sub_item['title'];
+      $variables['items'][$key]['items'][$key_sub_item]['path'] = $sub_item['url']->toString();
+      $variables['items'][$key]['items'][$key_sub_item]['attributes'] = new Attribute(['class' => $subclass]);
```

Comment:
@escuriola
This change seems to be pointless.
The original Drupal menu item does not have any 'items' key that we were destroying. 'items' is a completely new key that is only understood by bcl-navigation.